src/xcode/ENA/ENA/Source/Services/CoronaTestService/FamilyMemberCoronaTestService.swift

```diff
@@ -291,6 +295,9 @@ class FamilyMemberCoronaTestService: FamilyMemberCoronaTestServiceProviding {
 	
 	func reregister(coronaTest: FamilyMemberCoronaTest) {
 		coronaTests.value.append(coronaTest)
+		if !coronaTest.testResultWasShown {
+			badgeWrapper.increase(.familyMemberUnseenTests, by: 1)
+		}
```

Comment:
I think `.familyMemberUnseenTests` on the badge wrapper should be set to `unseenNewsCount` from the service every time the `coronaTests` publisher changes, not just increase by one if a test is restored from the recycle bin.

Reply:
yes that was wrong, I fixed it.